### ISSUE TITLE
Error upgrading from 0.6.0 to 0.6.1

### ISSUE THREAD

**Issue by ramonguiu** [timescale/promscale#800]
Hi, While upgrading from 0.6.0 to 0.6.1 , I am getting below error:-

`level=error ts=2021-10-12T05:05:31.770Z caller=runner.go:50 msg="aborting startup due to error" err="migration error: Error while trying to migrate DB: Error encountered during migration: error executing migration script: name versions/dev/0.5.2-dev/1-downsampling.sql, err ERROR: UPDATE requires a WHERE clause (SQLSTATE 21000)"`

I tried the workaround mentioned above , but the ALTER query is breaking:-

`ERROR:  column "table_schema" of relation "metric" does not exist`

Database version - PostgreSQL 13

Thanks

_Originally posted by @rinshadka in https://github.com/timescale/promscale/issues/755#issuecomment-940678655_

**Comment by paulfantom** [comment]
@rinshadka could you check the schema version with `select version from prom_schema_migrations;`?

**Comment by rinshadka** [comment]
Hi, Please see the results respectively,

```
 version
---------
 0.5.1
(1 row)

```
```
\dx
                                       List of installed extensions
    Name     |  Version  |   Schema   |                            Description
-------------+-----------+------------+-------------------------------------------------------------------
 plpgsql     | 1.0       | pg_catalog | PL/pgSQL procedural language
 promscale   | 0.1.2     | _prom_ext  | Promscale support functions
 timescaledb | 2.3.0-dev | public     | Enables scalable inserts and complex queries for time-series data
(3 rows)
```
Thanks

**Comment by rinshadka** [comment]
> 
> 
> Does the problem also occur when going from `0.5.1` -> `0.6.1` directly?

The issue is now resolved once I upgraded to `v0.6.2`, Thank you.
